### PR TITLE
Issue564 perez

### LINKED ISSUE #564: Perez implementation from Annex 60
We currently have models for 3 implementations of the perez model + other solar irradiance calculations. 
1) original version
2) copy from original version that was moved but not deleted
3) annex 60 version

I propose to use the Annex 60 version to avoid duplicate models.

### PR DESCRIPTION
**Pull request by Mathadon**
cleaned up BaseClasses with duplicates
using Annex60 base classes for solar irradiation
added unit tests for siminfomanager

this closes #564

### CODE REVIEW

**Comment by Mathadon** [comment]
@rubenbaetens  do you have time to review this?

**Comment by Mathadon** [comment]
I updated the unit tests again after a conflict. This can now be merged.

**Comment by damienpicard** [comment]
@Mathadon @rubenbaetens can this be merged? Or should I review it?

**Comment by Mathadon** [comment]
it should be reviewed first :)

**Comment by Mathadon** [comment]
bump

**Comment by rubenbaetens** [comment]
I agree with all changes here, and this can be merged for me. I'll let @Mathadon decide if we need to re-run the reference results ...

File: IDEAS/BoundaryConditions/Climate/Files.mo [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  model min5 "5-minute data"
+    extends IDEAS.BoundaryConditions.Climate.Files.Detail(filNam="_5.txt",
+        timestep=300);
+  end min5;
+
+  model min60 "60-minute data"
+    extends IDEAS.BoundaryConditions.Climate.Files.Detail(filNam="_60.txt",
+        timestep=3600);
+  end min60;
+
```
Comment (Mathadon) [inline IDEAS/BoundaryConditions/Climate/Files.mo]: I'm not sure where this comes from. This is probably some file that we removed already but that was still on my pc and accidentally recommitted.
Thanks for noticing!

**Comment by Mathadon** [comment]
I reran unit tests.

**Pull request merged**